NitroxClient/Logger/ClientLogger.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NitroxClient.Logger
+{
+    public class ClientLogger
+    {
+        [Flags]
+        public enum LogLevel
+        {
+            Disabled = 0,
+            InGameMessages = 1,
+            ConsoleMessages = 2,
+            ConsoleDebug = 4
+        }
+        private static LogLevel logLocations = LogLevel.Disabled;
+
+        // Set with combinaition of enum -- setLogLocation(LOG_CONSOLE | LOG_ERRORMESSAGE)
+        public static void SetLogLocation(LogLevel location)
+        {
+            ClientLogger.logLocations = location;
+            Console.WriteLine("Log location set to " + ClientLogger.logLocations);
+        }
+
+        public static void WriteLine(String msg)
```

Comment:
Since we don't have the ability to write anything other than a line I would recommend simplifying these method names.  I've seen loggers usually use Info() and Debug().